### DISCUSSION TITLE
Struggling : HTTP Methods during install

### DISCUSSION THREAD

**Discussion by twoex** [YetiForceCompany/YetiForceCRM#15454] [How to]
Hello,

I setup a VPS running ubuntu 20.04 and lamp... I resolved all the necessary requirements needed for YF but I am struggling now for days to figure out how to correct http methods and enable all the necessary options.. I have tried so hard, researched, but I cannot seem to figure this out at all.

Would anyone be willing to point me in the right direction as to how to get past this requirement?

Would truly be greatful and appreciated.

Thank you

**Comment by mariuszkrzaczkowski** [comment]
There was a bug in verifying HTTP methods, this has been fixed here https://github.com/YetiForceCompany/YetiForceCRM/commit/e94ffdb39536d01a640d871eec15fc52ff0bdab3

**Reply by twoex** [reply]
Dziękuję! Would make sense why it wasn't so easy to figure out... I updated the 2 files with the changes, although on the right hand column it doesn't show me what options I currently do have enabled... where would I go to enable these options on the system, if I may ask that..

**Reply by mariuszkrzaczkowski** [reply]
crm version?

**Reply by twoex** [reply]
> 

crm version? latest version of YetiForce, blank vps server, ubuntu 20.04... with lamp.. i tried putting the http methods in virtual hosts, htaccess, apace2.conf, etc and I cannot get it to accept changes for yetiforce to recognize it.. its when I choose the install option on own server... the verify section actually now shows an output, just shows OPTION
